client/components/Account/DeleteAccountForm.vue

```diff
@@ -11,12 +11,13 @@ export default {
       url: '/api/accounts',
       method: 'DELETE',
       loadAccount: true,
-      title: 'Delete account',
-      fields: [],
-      content: 'Deleting your account is permanent and irreversible. Proceed only if you understand these consequences.',
+      title: 'Delete shared account',
+      content: `Deleting the shared account will permanently and irreversibly delete all of ${this.$store.getters.accountName}'s health data.<br>\
```

Comment:
Does deleting a shared account also delete all the user accounts created under the shared account? If so, might be nice to add "...delete all of ...health data and all the user accounts, including yours. ..."

Reply:
Hmm, I feel like since this says it's deleting the shared account's data, it could be inferred that all the users are deleted? Would be nice to maybe have a another confirmation/warning even after clicking the button, though

Reply:
ok this is the distinction I have tried to make very clear but still I guess has not been articulated well enough:

we have no "user accounts" there is only shared accounts which have many "users" associated with them that are simply credentials (i.e. username/password pairs)

I know this language is somewhat confusing because people conflate "user" and "account" as meaning the same thing -- which is why I have tried to remove all mentions of "user" on this page.